cmd/livepeer/livepeer.go

```diff
@@ -633,10 +634,19 @@ func main() {
 				timeWatcher,
 				cfg,
 			)
+			n.SetTicketEV(ev)
 			if err != nil {
 				glog.Errorf("Error setting up PM recipient: %v", err)
 				return
 			}
+			mfv, _ := new(big.Int).SetString(*maxFaceValue, 10)
+			if mfv == nil {
+				glog.Errorf("-maxFaceValue must be a valid integer, but %v provided. Restart the node with a different valid value for -maxFaceValue", *maxFaceValue)
+				return
+			} else {
+				n.SetFaceValueLimit(mfv)
```

Comment:
For the sake of consistency, maybe let's name it `n.SetMaxFaceValue()` instead of `n.SetFaceValueLimit()`?